.github/workflows/test.yaml

```diff
@@ -34,26 +34,78 @@ jobs:
           echo $DOING_RELEASE
 
   lint:
-    name: Code lint
-    runs-on: ubuntu-22.04
+    name: Code lint on ${{ matrix.metadata.build }}
+    runs-on: ${{ matrix.metadata.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        metadata: [
+          { 
```

Comment:
I'm not a big fan of adding 4 more extra jobs, we already have a lot of them.

Can we add the clippy tests to some other jobs that are relatively fast anyway?

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3499/commits/6ceb8ff312b3754c87d1d8405e0e28341dcfd0ce